tests/integration/test_dashboards.py

```diff
@@ -135,3 +135,24 @@ def test_dashboard_deploys_dashboard_with_big_widget(ws, make_dashboard, tmp_pat
     sdk_dashboard = dashboards.deploy_dashboard(lakeview_dashboard, dashboard_id=sdk_dashboard.dashboard_id)
 
     assert ws.lakeview.get(sdk_dashboard.dashboard_id)
+
+
+def test_dashboard_deploys_dashboard_with_right_widget_order(ws, make_dashboard, tmp_path):
+    sdk_dashboard = make_dashboard(display_name="Order Key")
+
+    with (tmp_path / "dashboard.yml").open("w") as f:
+        f.write("display_name: Order Key\n"
```

Comment:
We were thinking about the following `dashboard.yml` format:

``` yaml
display_name: <dashboard name>

widgets:
  <widget name/id>: 
      order: 1
  <widget name/id>:
      order: 0
   <widget name/id>:
      order: 2
```

Maybe you could add that to the docs first so that we can give feedback about

Reply:
Widget id is by default the file stem